src/js/plugin-bridge/PluginSDK.js

```diff
@@ -264,15 +264,15 @@ const getSDK = function (pluginID, config) {
   }
 
   let SDK = new PluginSDKStruct({
-    config: config || {},
     addStoreConfig,
+    constants,
+    onDispatch,
+    pluginID,
+    config: config || {},
     dispatch: createDispatcher(pluginID),
-    Store: StoreAPI,
     Hooks: hooks,
     routingService: RoutingService,
```

Comment:
Wonder if we should provide the routing service as a module instead... See https://github.com/dcos/dcos-ui/pull/1363 for an example /cc @MatApple